README.md

```diff
@@ -1,2 +1,6 @@
 # phenopacket2prompt
 GA4GH Phenopacket to LLM prompt
+
+Prompts generated with this code base have been used for:
+- A benchmark of some large language models against the state of the art tool for differential diagnostics, [Exomiser](https://github.com/exomiser/Exomiser). The bottom line, Exomiser clearly outperforms the LLMs as per our [preprint](https://www.medrxiv.org/content/10.1101/2024.07.22.24310816v3), [data at this zenodo](https://doi.org/10.5281/zenodo.14008476).
```

Comment:
Opinion, but we should probably just link the papers and not "blow our own horn"